tetrateci/tsb-util/tsb_util.py

```diff
@@ -198,6 +198,7 @@ def gen_direct_specific_objects(
         virtualserviceName="bookinfo-virtualservice",  # need to change
         ns=namespaces["product"],
         gatewayName=namespaces["product"] + "-gateway",
+        destinationFQDN="productpage." + namespaces["product"] + ".tetrate.test.com",
```

Comment:
why tetrate.test.com? Shouldn't it be svc.cluster.local?

Reply:
right